pkg/jobrunaggregator/jobrunaggregatoranalyzer/pass_fail.go

```diff
@@ -339,8 +339,12 @@ func (a *weeklyAverageFromTenDays) checkPercentileDisruption(jobRunIDToAvailabil
 
 	for jobRunID, disruption := range jobRunIDToAvailabilityResultForBackend {
 		if float64(disruption.SecondsUnavailable) > threshold {
+			fmt.Printf("fail: jobrun=%s backend=%s observed=%ds threshold=%.2fs thresholdPercentile=%d\n", jobRunID, disruption.ServerName,
```

Comment:
you want to add these to the summary as well?

Reply:
Yeah scratch moving it up, It'll be nice to see it right in the test message instead of the build log. Working on it now.